### ISSUE TITLE
Mobile VO skips when moving quickly

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/molarity#223]
For https://github.com/phetsims/QA/issues/476 but I checked and saw that it can happen in sims like Ohm's Law too.
When moving the sliders quickly, if you swipe two fingers up to start reading from the beginning, before the previous alerts are done reading, VO will sometimes stop reading or read from the wrong location. If it doesn't read anything, or stops reading, and you then move back to a slider or other object, it will then sometimes start reading the scene description from the beginning or sometimes from somewhere in the middle of the description. Can be hard to get consistent results, but happens frequently enough that some version can be reproduced relatively easily.

**Comment by KatieWoe** [comment]
Update: Looked at Ohm's Law on iOS 12 VO and this issue did not seem to occur. The allert that you miss does interrupt the scene description briefly, but it then carries on with the description. This indicates that there has been some change to VO that caused this.

**Comment by jessegreenberg** [comment]
OK thanks @KatieWoe - these kind of gesture issues are likely a VO bug and out of our control but I will try to make sure.

**Comment by jessegreenberg** [comment]
I am getting this to happen in this basic HTML so it isn't anything about PhET sims.
```html
<!DOCTYPE html>
<html>
  <head>
    <meta charset="UTF-8">
    <title>TEST PAGE</title>
  </head>
  <body>
    <article>
      <h1>This is an article</h1>

      <p id="alert" aria-live="polite"></p>
      <input id='slider'type="range">
    </article>
  </body>
  <script type="text/javascript">
    let i = 0;
    document.getElementById( 'slider' ).addEventListener( 'change', () => {
      document.getElementById( 'alert' ).textContent = 'test' + i++;
    } );
  </script>
</html>
```

Adding to a11y quirk tracker.

While testing this I also found inconsistent behavior with the two finger swipe up. Sometimes it goes to the "back button" sometimes it just goes to top of web page. I haven't found any reason for differences there yet, but we have an issue about this open already.

**Comment by jessegreenberg** [comment]
Made note in document.

**Comment by KatieWoe** [comment]
Note for future: This can also end up interrupting the alerts from the sliders in a similar way. https://github.com/phetsims/QA/issues/477
